be/src/storage/rowset/rowset_writer.cpp

```diff
@@ -203,7 +203,10 @@ Status RowsetWriter::flush_segment(const SegmentPB& segment_pb, butil::IOBuf& da
         // 1. create segment file
         auto path = Rowset::segment_file_path(_context.rowset_path_prefix, _context.rowset_id, segment_pb.segment_id());
         // use MUST_CREATE make sure atomic
-        ASSIGN_OR_RETURN(auto wfile, _fs->new_writable_file(path));
+        WritableFileOptions opts{.sync_on_close = config::sync_tablet_meta,
+                                 .skip_fill_local_cache = false,
```

Comment:
skip_fill_local_cache is true for local tablet?

Reply:
This is unintended change, should be reverted.